dali/pipeline/operators/geometric/bb_flip.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/geometric/bb_flip.h"
+
+namespace dali {
+
+DALI_REGISTER_OPERATOR(BbFlip, BbFlip, CPU);
+
+
+DALI_SCHEMA(BbFlip)
+                .DocStr(R"code(Operator for horizontal flip (mirror) of bounding box.
```

Comment:
How about making the same wording as in https://github.com/NVIDIA/DALI/pull/110/:
```
Tensor (`m` * `[x, y, w, h] or `m` * [left, top, right, bottom]`)
```